Formula/m-cli.rb

```diff
@@ -0,0 +1,23 @@
+class MCli < Formula
+  desc "Swiss Army Knife for Mac OS X"
+  homepage "https://github.com/rgcr/m-cli"
+  url "https://github.com/rgcr/m-cli/archive/v0.0.9.tar.gz"
+  sha256 "957e86c03a8196f60c852672aa0a7c86d9093347e37c16783e67e423fa56f341"
+
+  def install
+    prefix.install Dir["*"]
+    inreplace "#{prefix}/m", /^\[ -L.*|^\s+\|\| pushd.*|^popd.*/, ""
```

Comment:
Thanks for your explanation in the main thread. That sounds fine. Would you mind tweaking it a touch here:
```ruby
inreplace prefix/"m" do |s|
  s.gsub! /^\[ -L.*|^\s+\|\| pushd.*|^popd.*/, ""
  s.gsub! /MPATH=.*/, "MPATH=#{prefix}"
end
```

And adding a short explanation above the `inreplace` like:
```ruby
# Use absolute rather than relative path to plugins.
```
or such.